src/state/machines/turn.ts

```diff
@@ -166,6 +167,7 @@ export const TurnMachine = setup({
 
         const drawnCard = player.deck.shift();
         if (drawnCard) player.hand.push(drawnCard);
+        player.hand = [...player.hand].sort(getCardComparator(context.deck.ordering));
```

Comment:
I still want to argue this point😂 I think it will be better to do the sorting in the positioner. That way if we add more actions to turn machine we dont have to keep remembering to preserve the order. In my mind the sorting is a positional concern and not a state concern. It carries no logical value therefore I think the positioner should decide how and when things should be sorted

Reply:
I understand what you mean. However, if we handle sorting within the positioner, we lose control over when the sorting actually occurs. Currently, we can control when cards should be forced and when they shouldn’t. Sorting the player row every time it changes isn’t ideal, in my opinion – it could lead to too much stuff flying around on the screen at times.